tests/fixtures/404.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Should this be
```
<html lang="<%= htmlWebpackPlugin.options.lang %>>" 
```

Reply:
it doesn't need to be, this file is used to test internationalization.